### PR TITLE
pagerduty: Show on-call information when not on-call

### PR DESCRIPTION
**Pull request by drudge**
This PR tweaks the overall design of the PagerDuty app.

### Changes

* Adds a new view of the status bar that scrolls through the on-call status by team if the user is off-duty. 
* Changes the way I build the PagerDuty logo, fixes the margins, and allows for hiding it completely.
* Renamed triggered alert count label from "TRIG" to "NEW"
* Adds the ability to hide the on-call bar when you are not on-call

### Before this PR:
pagerduty

### After this PR:
pagerduty

### Notes

* I'm using the `humanize` module in Pixlet so this may require a new version to be published before it can be deployed
* I've included a devcontainer that allows developing Community apps entirely from Github Codespaces.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.222.0/containers/go/.devcontainer/base.Dockerfile
```
Comment (betterengineering) [inline .devcontainer/Dockerfile:1]: This is rad! Can we add it to a separate PR?

File: apps/pagerduty/pagerduty.star [hunk 2]
```diff
@@ -6,8 +6,8 @@ Author: drudge
 """
 
 load("cache.star", "cache")
-load("encoding/base64.star", "base64")
 load("encoding/json.star", "json")
+load("humanize.star", "humanize")
```
Comment (betterengineering) [inline apps/pagerduty/pagerduty.star:10]: Getting a release of pixlet cut so your tests pass! One sec

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (drudge) [reply .devcontainer/Dockerfile]: Sure thing, I've reverted my merge and I'll open a separate PR.

Comment (drudge) [reply .devcontainer/Dockerfile]: https://github.com/tidbyt/community/pull/174

**Comment by betterengineering** [comment]
Going to land this, it should pass on the main branch

**Pull request merged**